src/client/playwright.ts

```diff
@@ -59,6 +60,8 @@ export class Playwright extends ChannelOwner<channels.PlaywrightChannel, channel
 
     this._selectorsOwner = SelectorsOwner.from(initializer.selectors);
     this.selectors._addChannel(this._selectorsOwner);
+
+    this._channel.on('incomingSocksSocket', ({socket}) => SocksSocket.from(socket));
```

Comment:
Let's save the ports on Playwright object, so that we can validate dstPort on the client side as well.